### DISCUSSION TITLE
Video player

### DISCUSSION THREAD

**Discussion by zebramonkeylizard** [OnionUI/Onion#497] [Ideas]
Is there a way to play movies on the Miyoo Mini? I've read MPlayer is a common choice for some other handhelds like the V90. I'd like to watch TMNT (87-96) and Batman: The Animated Series (92-95) between gaming sessions.

**Comment by dwmccoy** [comment]
Yes you can play 480p MP4 files with no issue.  You need a copy of ffplay and a copy of parasyte.  Both of those apps can be found here https://github.com/steward-fu/miyoo-mini/releases/tag/stock  .  The ffplay application should be placed into the /EMU folder of the sdcard and the parasyte application should be placed into the /APP folder of the sdcard.  Create a folder on the root of the sdcard called VIDEO.  Place your 480p mp4 files into the /VIDEO folder.  The ffplay application will be located in the consoles  menu once installed.  You will need to edit the "extlist" of the /EMU/ffplay/config.json file to be able to see the videos in the ffplay list.  The config.json file should be configured like below

{
"label":"FFPlay",
"icon":"icon.png",
"iconsel":"",
"launch":"launch.sh",
"rompath":"../../VIDEO",
"imgpath":"../../Imgs/VIDEO",
"useswap":1,
"shortname":0,
"hidebios":0,
"extlist":"mp4|MP4"
}

**Reply by zebramonkeylizard** [reply]
Thanks. It looks like this has been available for a while. @Sichroteph I hope this can be added to Onion OS.

**Reply by zebramonkeylizard** [reply]
I have this working now, but the audio is very low using the speaker.

**Reply by dwmccoy** [reply]
I would agree that the sound is low.  I think it's a limitation of the speaker.  You could use video conversion tool "Handbrake" to increase the gain of the audio.  I think I read somewhere that "Eggs" Custom Keymon located here https://github.com/jimgraygit/Onion/wiki/6.-Miyoo-mini-apps#custom-keymon-credit-eggs will allow you to overdrive the volume of the speaker but I have not tried it.

**Comment by schmurtzm** [comment]
Video player is available in Onion, I close this one.
https://onionui.github.io/docs/apps/video-player

(use [volume boost shortcut](https://onionui.github.io/docs/apps/shortcuts) if the sound is low)
